src/Microsoft.AspNet.Mvc.Razor/RazorView.cs

```diff
@@ -9,6 +9,7 @@
 using Microsoft.AspNet.Mvc.Rendering;
 using Microsoft.AspNet.PageExecutionInstrumentation;
 using Microsoft.Framework.Internal;
+using Microsoft.Framework.WebEncoders;
```

Comment:
Changes to this class and `RazorViewFactory` are downstream of the switch from `[RazorInject]`. If required because property injection wasn't working (and the encoder is used more now), fine.